src/pure/host.mli

```diff
@@ -149,5 +139,5 @@ val markup: t -> Display_markup.t
 val execution_timeout: t -> Time.t option
 (** The execution timeout configured for the host. *)
 
-val connection: t -> [ `Localhost | `Ssh of Ssh.t | `Named of string ]
+val connection: t -> [ `Localhost | `Ssh of Ssh.t ]
```

Comment:
I still use `` `Ssh _`` for some stuff; then when an alternative raises and outperforms it (maybe part of coclobas, maybe independent), we'll be able to remove it.